traits/src/liquidity_mining.rs

```diff
@@ -185,3 +186,13 @@ pub trait PriceAdjustment<GlobalFarm> {
 pub trait Inspect<AccountId> {
 	fn pot_account() -> Option<AccountId>;
 }
+
+pub trait XykAddLiquidity<Origin, AssetId, Balance> {
```

Comment:
Why is this in LM trait? This should be in AMM/xyk trait

Reply:
since adding liquidity requires different params for pools XYK and LBP (which implements AMM trait), therefore I created a separate one for XYK.